cli/ceph/osd/pool.py

```diff
@@ -0,0 +1,122 @@
+from cli import Cli
+
+
+class Pool(Cli):
+    """This module provides CLI interface to manage the MGR service."""
+
+    def __init__(self, nodes, base_cmd):
+        super(Pool, self).__init__(nodes)
+
+        self.base_cmd = f"{base_cmd} pool"
+
+    def rename(self, current_pool_name, new_pool_name):
+        """
+        To rename a pool
+        Args:
+            current_pool_name (str): Existing pool name
+            new_pool_name (str): New name to be assigned
+        """
+        cmd = f"{self.base_cmd} rename {current_pool_name} {new_pool_name}"
+        out = self.execute(sudo=True, cmd=cmd)
+        if isinstance(out, tuple):
+            return out[0].strip()
+        return out
+
+    def delete(self, pool_name):
+        """
+        To delete a pool
+        Args:
+            pool_name (str): Name of the pool to be deleted
+        """
+        cmd = f"{self.base_cmd} delete {pool_name}"
+        out = self.execute(sudo=True, cmd=cmd)
+        if isinstance(out, tuple):
+            return out[0].strip()
+        return out
+
+    def set_quota(self, pool_name, max_objects=None, max_bytes=None):
+        """
+        To set pool quotas for the maximum number of bytes and the
+        maximum number of objects per pool
+        Args:
+            pool_name (str): Name of the pool to be deleted
+            max_objects (str): object count
+            max_bytes (str): bytes count
+        """
+        cmd = f"{self.base_cmd} set-quota {pool_name}"
+        if max_objects:
```

Comment:
Need to rework here. In case both `max_objects` & `max_bytes` are `None`, cmd will set `max_bytes` to `None`

Reply:
Done